fields/ModuleNavigationField.php

```diff
@@ -61,10 +61,9 @@ protected function renderStatic($entry)
             }
 
             $disabledLink = $this->courseManager->isModuleDisabledLink($learner, $course, $module);
-            // TODO implement getNextActivity for a learner, for the moment choose the first activity of the module
```

Comment:
[je passe rapidement sur ce truc de cette 3ème PR]

Il y a d'ailleurs ma fermeture de commentaire qui reste. J'aurais aimé que tu laisse le commentaire car c'est un bout de code qui est normalement commun avec ModuleNavigationField et je n'avais pas pris le temps de le factoriser quand j'avais refactoré le code de Flo. 

Et surtout je me rend compte que le code n'est plus le même, et pourtant l'affichage du bouton doit être le même !

Reply:
J'ai remis le commentaire https://github.com/YesWiki/yeswiki-extension-lms/commit/6309eae425f2ac5497562400d261b1766a3a18b7#diff-92b3df2a1104d8d7a1a707c354aea7a07a53f6dbc47c7c3ef01a916f5625120cR59

Reply:
Il y a le commentaire ` // TODO implement getNextActivity for a learner, for the moment choose the first activity of the module` qui me semble plus d'actualité, non ?

Mais sinon pour moi il y a un vrai soucis ici, tu as remis le commentaire qui dit qu'une factorisation sera faite mais le code n'est plus le même entre $courseController->renderModuleCard et $moduleNavigationField->renderStatic, donc pour moi ce n'est pas clair à ce niveau car on est censé avoir le même bouton qui s'affiche.

Reply:
Je te laisse clarifier ceci à ta guise...